modules/viewing-the-image-pull-source.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="viewing-the-image-pull-source_{context}"]
+= Viewing the image pull source
+
+You can view the source of your pulled images by using the CRI-O logs on your nodes. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* Review the CRI-O logs on a node or for the worker nodes:
```

Comment:
It's better to use "Review the CRI-O logs on a master node or for the worker nodes", for your example use a master node. But it's a tiny problem, you can decide modify it or not.